application/src/test/java/org/opentripplanner/routing/core/TemporaryVerticesContainerTest.java

```diff
@@ -63,6 +59,7 @@ public void temporaryChangesRemovedOnClose() {
     subject = new TemporaryVerticesContainer(
       g,
       TestVertexLinker.of(g),
+      id -> Set.of(),
```

Comment:
```suggestion
      id -> List.of(),
```
We have a convention to use lists over sets, so seeing `Set` here make ma ask why. Since the API take a collection both work, but `List.of` is preferable. This apply to a few other places.